tests/bsim/bluetooth/mesh/src/test_provision.c

```diff
@@ -1739,35 +1759,50 @@ static void test_device_pb_remote_server_ncrp_second_time(void)
 }
 #endif /* IS_RPR_PRESENT */
 
-#define TEST_CASE(role, name, description)                                     \
-	{                                                                      \
-		.test_id = "prov_" #role "_" #name, .test_descr = description, \
-		.test_post_init_f = test_##role##_init,                        \
-		.test_tick_f = bt_mesh_test_timeout,                           \
-		.test_main_f = test_##role##_##name,                           \
-		.test_delete_f = test_terminate                                \
-	}
-#define TEST_CASE_WBACKCHANNEL(role, name, description)                                     \
-	{                                                                      \
-		.test_id = "prov_" #role "_" #name, .test_descr = description, \
-		.test_post_init_f = test_##role##_init,                        \
-		.test_pre_init_f = test_back_channel_pre_init,                 \
-		.test_tick_f = bt_mesh_test_timeout,                           \
-		.test_main_f = test_##role##_##name,                           \
-		.test_delete_f = test_terminate                                \
-	}
+/* Test cases by default will run over PB_ADV*/
+#define TEST_CASE(role, name, description)                                                         \
+	{.test_id = "prov_" #role "_" #name,                                                       \
+	 .test_descr = description,                                                                \
+	 .test_post_init_f = test_##role##_init,                                                   \
+	 .test_tick_f = bt_mesh_test_timeout,                                                      \
+	 .test_main_f = test_##role##_##name,                                                      \
+	 .test_delete_f = test_terminate}
+/* Test cases that will run over either PB_ADV or PB_GATT */
+#define TEST_CASE_BEARER(role, bearer, name, description)                                          \
+	{.test_id = "prov_" #role "_" #bearer "_" #name,                                           \
+	 .test_descr = description,                                                                \
+	 .test_post_init_f = test_##role##_init,                                                   \
+	 .test_tick_f = bt_mesh_test_timeout,                                                      \
+	 .test_main_f = test_##role##_##name,                                                      \
+	 .test_delete_f = test_terminate}
+/* Test cases that will run over either PB_ADV or PB_GATT */
+#define TEST_CASE_WBACKCHANNEL(role, bearer, name, description)                                    \
```

Comment:
Adding `bearer` param and using it only to create test id is functionally redundant however I think it helps readability. But I am not against removing completely since upper most test case name has bearer names, anyways.